vars/getCheckoutInfo.groovy

```diff
@@ -0,0 +1,32 @@
+def call(scmVars) {
+  def repoInfo = [:]
+
+  repoInfo.branch = env.ghprbSourceBranch ?: scmVars.GIT_LOCAL_BRANCH ?: scmVars.GIT_BRANCH
```

Comment:
I think this is going to work because we're only relying on the scmVars when we're not using `ghprb`